scipy/spatial/qhull_src/src/libqhull_r.h

```diff
@@ -37,6 +37,7 @@
 #include <limits.h>
 #include <time.h>
 #include <stdio.h>
+#include <string.h>
```

Comment:
@rlucas7 I've pushed in the larger patch that was recommended. I confirmed locally on a Mac with XCode 12 that there was a failure before/pass after applying this changeset.

However, I believe way may not need the qhull-related shims since I think I already backported something @pv reviewed a few weeks ago about this.

Maybe I'll find some time to try trimming this off the patch and seeing if a clean build still works..

Reply:
> @rlucas7 I've pushed in the larger patch that was recommended. I confirmed locally on a Mac with XCode 12 that there was a failure before/pass after applying this changeset.

@tylerjereddy  Good to hear that is a possible fix. 

> However, I believe way may not need the qhull-related shims since I think I already backported something @pv reviewed a few weeks ago about this.

I think when I tested this locally I had a branch off the maintenance branch 1.5.x so it already had some of the patches from other merged PRs that partially fixed. 
I didn't check the diff of the the maintenance/1.5.x branch against master though so there might be other stuff needed too. 
When I was suggesting the other patch I was referring to this one:
https://git.sagemath.org/sage.git/tree/build/pkgs/scipy/patches/extern_decls.patch?id=711fe05025795e44b84233e065d240859ccae5bd
which is linked here:
https://github.com/scipy/scipy/issues/12860#issuecomment-695172570
it looks like parts (maybe all?) of that patch would fix and it's a little more comprehensive than this one :) 
not sure if that's the patch you're looking at or not.  

> Maybe I'll find some time to try trimming this off the patch and seeing if a clean build still works.

Yeah I'm pretty busy with work this week, I might get some time the upcoming weekend on Sunday to take a look but depends on if I can get other things done quickly.

Reply:
Uhh, that is the patch that's applied here, yes. I'm saying we may not need the qhull part of it, that is all.

Reply:
The qhull part was gh-12864

Reply:
I didn't notice at the time that the PR closed the issue, even though some changes were left out, sorry.

Reply:
> Uhh, that is the patch that's applied here, yes. I'm saying we may not need the qhull part of it, that is all.

ok makes sense.